src/elements/dialog/dialog/dialog.component.ts

```diff
@@ -146,6 +157,26 @@ class SbbDialogElement extends SbbOverlayBaseElement {
     }
   }
 
+  private _detectIntermediateElement(): void {
+    const potentialChildElements = ['sbb-dialog-title', 'sbb-dialog-content', 'sbb-dialog-actions'];
+
+    const hasOnlyOneDirectChild =
+      Array.from(this.children).filter((el) => el instanceof HTMLElement).length === 1;
+    const noExpectedElementAsDirectChild = potentialChildElements.every(
+      (name) => this.firstElementChild?.localName !== name,
+    );
+    const intermediateElementHasExpectedChildren = Array.from(
```

Comment:
Are we sure that the children are already attached when the intermediate element is slotted?

Reply:
only use noExpectedElementAsDirectChild